src/routes/todos/+page.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
-	import { enhance, type SubmitFunction } from '$app/forms';
-	import type { ActionData, PageData } from './$types';
+	import { enhance } from '$app/forms';
+	import type { ActionData, PageData, SubmitFunction } from './$types';
 
```

Comment:
this should get rid of the errors
```suggestion
	import { enhance } from '$app/forms';
	import type { ActionData, PageData, SubmitFunction } from './$types';

```